catch-all.scala:4: warning: This catches all Throwables. If this is really intended, use `case _ : Throwable` to clear this warning.
  try { "warn" } catch { case _ => }
                              ^
catch-all.scala:6: warning: This catches all Throwables. If this is really intended, use `case x : Throwable` to clear this warning.
  try { "warn" } catch { case x => }
                              ^
catch-all.scala:8: warning: This catches all Throwables. If this is really intended, use `case x : Throwable` to clear this warning.
  try { "warn" } catch { case _: RuntimeException => ; case x => }
                                                            ^
3 warnings
error: No warnings can be incurred under -Werror.
